src/buf/iter.rs

```diff
@@ -28,10 +28,10 @@ impl<T> IntoIter<T> {
     /// # Examples
```

Comment:
Thanks. It could make sense to add a paragraph like this.
```suggestion
    /// Using this constructor is equivalent to calling `into_iter`.
    ///
    /// # Examples
```